addons/website/static/tests/tours/grid_layout.js

```diff
@@ -46,3 +46,41 @@ wTourUtils.registerWebsitePreviewTour('website_replace_grid_image', {
     },
     ...wTourUtils.clickOnSave()
 ]);
+
+wTourUtils.registerWebsitePreviewTour("scroll_to_new_grid_item", {
+    test: true,
+    url: "/",
+    edition: true,
+}, () => [
+    // Drop enough snippets to scroll.
+    wTourUtils.dragNDrop({id: "s_text_image", name: "Text - Image"}),
+    wTourUtils.dragNDrop({id: "s_image_text", name: "Image - Text"}),
+    wTourUtils.dragNDrop({id: "s_image_text", name: "Image - Text"}),
+    // Toggle the first snippet to grid mode.
+    wTourUtils.clickOnSnippet({id: "s_text_image", name: "Text - Image"}),
+    wTourUtils.changeOption("layout_column", 'we-button[data-name="grid_mode"]'),
+    // Add a new grid item.
+    wTourUtils.changeOption("layout_column", 'we-button[data-add-element="image"]'),
```

Comment:
Why you don't test the flow with a button or a text instead ? That way you don't have to create the image attachment and add step to select the image in the media dialog ? :thinking:

Reply:
That's what I wanted to do at first, but I decided to keep the image so we are sure that the entire flow is working (I mean this is the most complex one so better it be tested) :slightly_smiling_face: 
(it is the test that indicated me that it was not working well so I think it is a good thing to have it like that)